core/server/pom.xml

```diff
@@ -32,6 +32,10 @@
 
   <dependencies>
     <dependency>
+      <groupId>com.google.protobuf</groupId>
```

Comment:
we should always include direct imports in the module pom, rather than relying on the indirect dependencies such as alluxio-core-common